.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+<!-- Please include a detailed summary of the changes made. Also please link the issue that this PR is fixing. -->
+
+Fixes # (issue)
+
+<!-- Please place an x in the [ ] to check off the type of change for this PR -->
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Chore items (this includes basic clean up of files, or updates to packages)
+- [ ] Updates to documentation
+
+<!-- Please make sure to go through the entire checklist before requesting a review. Place an x in the [ ] to check off all of the items completed -->
+
+## Checklist
+
+- [ ] I have a descriptive title for my PR
+- [ ] I have linked the issue to this [PR](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
```

Comment:
Do we need the word PR wrapped up into a link like this?
I would rewrite this line:
```
- [ ] I have linked this PR to the corresponding issue. [See how to do that here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
```

Reply:
I am not sure if we need it that way or not, maybe @jdwilkin4 can clarify our confusion about this.

Reply:
ok, waiting then

Reply:
Pull request and PR are the same thing so either works fine. 
If you want to go with pull request that is cool.

This is not a standardized checklist. 
Everyone does it differently.

Reply:
Ok. And how about the change of the linked text which I requested? I would like to see line 22 as I described above

Reply:
@sparkO7 I still think that my way to write the phrase and link(line 22) is more clear:

first goes the instruction itself, just text
second goes link, in case somebody needs this link

In the original version it's all mixed up a little, not a fan of that

As for `PR` or `pull request` - it's not a big deal, as said above